subsys/bluetooth/controller/hci_internal.c

```diff
@@ -358,6 +358,14 @@ static void supported_commands(sdc_hci_ip_supported_commands_t *cmds)
 
 	cmds->hci_le_set_connectionless_cte_transmit_parameters = 1;
 	cmds->hci_le_set_connectionless_cte_transmit_enable = 1;
+#endif
+
+#if defined(CONFIG_BT_CTLR_DF_CONN_CTE_RSP)
+	cmds->hci_le_connection_cte_response_enable = 1;
```

Comment:
cmds->hci_le_transmitter_test_v3 is required by both CONFIG_BT_CTLR_DF_ADV_CTE_TX and CONFIG_BT_CTLR_DF_CONN_CTE_RSP.